src/app/app.routes.ts

```diff
@@ -1,3 +1,17 @@
 import { Routes } from '@angular/router';
+import { AppComponent } from './app.component';
+import { Component } from '@angular/core';
+import { LoginComponent } from './login/login.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+  {
+    path: '',
+    component: AppComponent,
+    children: [
+      {
+        path: '',
```

Comment:
add a path such that only login component is called when /login route is called. and not for  home path